android/app/build.gradle

```diff
@@ -26,8 +26,9 @@ android {
             enable true
 
             reset ()
-            include = []
-            include.addAll("${liboAndroidAbi}".split(' ').collect{it as String})
+            def includedAbis = []
+            includedAbis.addAll("${liboAndroidAbi}".split(' ').collect{it as String})
+            include includedAbis
```

Comment:
If it's just a fixup, best to git rebase -i --autosquash, and force-push :-)

Reply:
Agreed. I'm aiming to minimise the force-pushes.